xbmc/pvr/guilib/guiinfo/PVRGUIInfo.cpp

```diff
@@ -428,6 +428,13 @@ bool CPVRGUIInfo::GetListItemAndPlayerLabel(const CFileItem* item, const CGUIInf
         strValue = recording->IsRadio() ? m_strPlayingRadioGroup : m_strPlayingTVGroup;
         return true;
       }
+      case VIDEOPLAYER_YEAR:
+      case LISTITEM_YEAR:
+        if (recording->GetPremiered().IsValid())
+        {
+          strValue = StringUtils::Format("%i", recording->GetPremiered().GetYear());
+        }
+        return true;
```

Comment:
`return false;`

See line 433 ff.

Reply:
> 
> 
> `return false;`
> 
> See line 438 ff.

If I return false there it continues to VideoGUIInfo tag logic as it does now and it will get filled with the first aired date.year.  I return true to leave it empty

Reply:
So, you want it to return nothing in case the recording has no valid premiered year?

What is the semmanticd of LISTITEM_YEAR, then. It should not be sometimes first aired, sometimes premiered.

For me this looks like we should have one guiinfo label for first aired and another one for premiered.

Reply:
That is the purpose of the PR.  If you look at the "before" video every title get the year from first aired when it should only be the year when it is expressly given by SetYear()  specifically for movies and specific shows that are "new" series.  If you go to a scraping site like  thetvdb  you can search common examples

```
Battlestar Galactia 
Battlestar Galactia (2003)
Battlestar Galactia (2020)

Doctor Who
Doctor Who (2005)

The Twilight Zone
The Twilight Zone (1985)
The Twilight Zone (2002)
The Twilight Zone (2019)
```
and each series has unique S/E numbering.   When there is only on series the year is rarely appended.

Also the year is not appended in the EPG  so I feel it should be consisten

Reply:
Would it not be simpler for PVRRecording to override `GetYear()` so it only returns the premiered year if available?

Would there be any side effects to doing that?

Reply:
> 
> 
> Would it not be simpler for PVRRecording to override `GetYear()` so it only returns the premiered year if available? The intent would be much clearer.
> 
> Would there be any side effects to doing that?

In this case recording is a sub class of CVideoInfoTag so GetYear() will return First Aired by preference.  I will check for any issue.

Reply:
PVRGUIInfo no longer requires change